configurations/create_imaqal_database.py

```diff
@@ -0,0 +1,187 @@
+                    ],
+                    ws_code_string_value="gender"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="IMAQAL_household_language",
+                    engagement_db_dataset="household_language",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("household_language"), auto_coder=None)
+                    ],
+                    ws_code_string_value="hh language"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="IMAQAL_location",
+                    engagement_db_dataset="location",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("mogadishu_sub_district"),
+                                                auto_coder=somali.DemographicCleaner.clean_mogadishu_sub_district),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("somalia_district"), auto_coder=lambda text:
+                                                somali.DemographicCleaner.clean_somalia_district(text)
+                                                if somali.DemographicCleaner.clean_mogadishu_sub_district == Codes.NOT_CODED else Codes.NOT_CODED),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("somalia_region"), auto_coder=None),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("somalia_state"), auto_coder=None),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("somalia_zone"), auto_coder=None),
+                    ],
+                    ws_code_string_value="location"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="IMAQAL_recently_displaced",
+                    engagement_db_dataset="recently_displaced",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("recently_displaced"), auto_coder=somali.DemographicCleaner.clean_yes_no)
+                    ],
+                    ws_code_string_value="recently displaced"
+                ),
+            ],
+            ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+            project_users_file_url="gs://avf-project-datasets/2022/IMAQAL-POOL/coda_users.json"
+        )
+    ),
+    rapid_pro_target=RapidProTarget(
+        rapid_pro=RapidProClientConfiguration(
+            domain="textit.com",
+            token_file_url="gs://avf-credentials/imaqal-text-it-token.txt"
+        ),
+        sync_config=EngagementDBToRapidProConfiguration(
+            normal_datasets=[
+                DatasetConfiguration(
+                    engagement_db_datasets=["age"], 
+                    rapid_pro_contact_field=ContactField(key="engagement_db_age", label="DB Age")
```

Comment:
```suggestion
                    rapid_pro_contact_field=ContactField(key="imaqal_engagement_db_age", label="imaqal engagement db age")
```
to allow flexibility if we ever want to sync other workspaces fields.